.github/workflows/test-e2e.yml

```diff
@@ -43,22 +43,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Download the build folder
-        uses: actions/download-artifact@v3
-        with:
-          name: build
-          path: assets
+      # - name: Download the build folder
+      #   uses: actions/download-artifact@v3
+      #   with:
+      #     name: build
+      #     path: assets
 
       - name: Cypress run
         uses: cypress-io/github-action@v5
         with:
           record: true
           parallel: true
-          start: npm start
+          start: npm run docker:start
           config-file: cypress.config.js
-          # wait for 2 minutes for the server to respond
-          wait-on: 'http://localhost:1337'
-          wait-on-timeout: 120
+          config: baseUrl=http://localhost:8080
+          # wait for 5 minutes for the server to respond
+          wait-on: 'http://localhost:8080'
```

Comment:
Hard coded the CI e2e to point to localhost:8080 since the config file inherits the cypress BASE_URL from the VITE_API_URL when available. Otherwise atm this would try to run the tests against localhost:1337, which failed because the client wasn't built or served on it. I tried running the build step in the `server` service config before booting the backend, but it was crashing the service for some reason.

I think ultimately that running the tests against a built version of the client, served by the `server` would be a better representation of the real application (that's how we've been doing it). But to make it work we'd need to get the `server` service to actually build the app